### ISSUE TITLE
the "Go to resource" link is (often) not very useful for NDE datasets

### ISSUE THREAD

**Issue by EnnoMeijers** [CLARIAH/ineo-collaboration#6]
Due to the ongoing implementation work for NDE datasets the links to the landingpages of the datasets are very different in quality and usefulness, they often do not show detailed information like the available distributions and their mime types. To offer a better service for the end user pointing to the landingpage of the NDE-datasetregister would more useful.

**Comment by menzowindhouwer** [comment]
Isn't this a metadata quality problem?  I prefer to link to the resource page of the data provider not the aggregator. Do you see a pattern when this is useless?

Is there actually a landing page for a dataset in the NDE-datasetregister? Or did you mean by "the landingpage of the NDE-datasetregister" just https://datasetregister.netwerkdigitaalerfgoed.nl/?

**Comment by EnnoMeijers** [comment]
I agree that data providers should provide a human readable landing page for each dataset they provide but sadly this is not yet the case. I hope this will improve in the coming years due to the NDE data quality efforts. 

The NDE-datasetregister provides a landing page that can be accessed using: `https://datasetregister.netwerkdigitaalerfgoed.nl/show.php?uri=<URI of the dataset>`. This could be added to the sparql query to create an addition variable that denotes the datasetregister landing page. It could of course also be added as a hard coded link for the "Go to resource" link.

**Comment by EnnoMeijers** [comment]
the problem is no longer there

**Comment by menzowindhouwer** [comment]
@EnnoMeijers did you really meant to close this issue? or was it mean to be #4?

**Comment by EnnoMeijers** [comment]
@menzowindhouwer thanks for noticing, I closed the wrong one indeed

**Comment by EnnoMeijers** [comment]
Hi Menzo, is any work planned for this issue? With the limited information currently presented within INEO, linking to the Datasetregister's landing page is, for the timing being, a necessary step to provide the user with actionable information. I would like to present the results of our work through NDE's communication channels but the issue is holding me back.

**Comment by menzowindhouwer** [comment]
Hi Enno, I still prefer not to link back to the NDE aggregator but to the LandingPage given by the provider. We could still do so temporarily for marked bad LandingPages, while preferably there is a parallel NDE proces to improve the quality. We could let that be managed by a github resource that NDE can maintain via PRs. In general I don't want to refer back to the aggregator by default. The default should stay the LandingPage ...
